tests/unit/test_catalog_loader.py

```diff
@@ -24,37 +23,34 @@ def __init__(self, **kwargs):
         self.kwargs = kwargs
 
 
-def test_get_id_generator():
-    db = SQLiteDatabaseEngine.from_db_file(":memory:")
-
-    id_generator = SQLiteIDGenerator(db, table_prefix="prefix")
-    assert id_generator.db == db
+def test_get_id_generator(sqlite_db):
```

Comment:
[Unrelated] - are these tests run in the Studio test suite?

Reply:
As far as I can tell, yes they are run - see the run here for example: https://github.com/iterative/datachain/actions/runs/10258065848/job/28381752999?pr=229

We probably could add a skip for these SQLite-specific tests, but that should probably be in a followup.